flag.go

```diff
@@ -828,19 +830,24 @@ func (f *FlagSet) usage() {
 
 func (f *FlagSet) setFlag(flag *Flag, value string, origArg string) error {
 	if err := flag.Value.Set(value); err != nil {
-		return f.failf("invalid argument %q for %s: %v", value, origArg, err)
+		return f.failf("invalid argument %q for %v: %v", value, origArg, err)
+	}
+	if !flag.isSetMultipleTimes && flag.Changed && !strings.Contains(flag.Value.Type(), "Slice") {
+		flag.isSetMultipleTimes = true
+		fmt.Fprintf(os.Stderr, "Flag %q is set multiple times. Using last value\n", flag.Name)
 	}
+
 	// mark as visited for Visit()
 	if f.actual == nil {
 		f.actual = make(map[NormalizedName]*Flag)
 	}
 	f.actual[f.normalizeFlagName(flag.Name)] = flag
 	f.orderedActual = append(f.orderedActual, flag)
 	flag.Changed = true
-	if len(flag.Deprecated) > 0 {
+	if len(flag.Deprecated) > 0 && !flag.isSetMultipleTimes {
```

Comment:
Yes, I also thought about what. pflag will warn multiple times about deprecated flag, if user sets deprecated flag multiple times. 
Should I add this part in separate PR?